app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, file, new_file = command.split(" ", maxsplit=3)
```

Comment:
Setting maxsplit=2 means the string will be split into at most three parts, which aligns with your expectation.

Reply:
but he maxsplit parameter specifies the maximum number of splits, not the exact number. If the actual number of possible splits in the string is less than maxsplit, the method will perform as many splits as possible.

Reply:
so its better to perform an additional check using len()  to make sure that the command has exactly three parts